README.md

```diff
@@ -26,52 +30,24 @@ You will have been added to a private discord channel with a few members of our
 
-export default function SomePage({ data }) {
-	return <p>{JSON.stringify(data)}</p>
-}
+We will provide you with a `DATO_API_TOKEN` for use in the `.env`, and for Sr. candidates also in the `createDB.ts`, file so that you can load data from a Dato instance. Also the cda-explorer.datocms.com tool we reference below will also need you to put in your `DATO_API_TOKEN` for it to work.
 
-export async function getStaticProps() {
-	const data = await rivetQuery({ query })
-	return { props: { data } }
-}
-```
-
-You'll see a similar template provided in the `pages/people` folder, which is working out of the box.
-
-Our CMS, [DatoCMS](https://www.datocms.com/), exposes a public GraphQL API to our data which [you can explore here using GraphiQL](https://cda-explorer.datocms.com/?apitoken=dc45ff8c8b27dd22a7c24aaaf8aa75&query=query%20%7B%0A%20%20allDepartments%28first%3A%20100%29%20%7B%0A%20%20%20%20name%0A%20%20%20%20parent%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%7D%0A%20%20%0A%20%20allPeople%28first%3A%20100%29%20%7B%0A%20%20%20%20name%0A%20%20%20%20avatar%20%7B%0A%20%20%20%20%20%20url%0A%20%20%20%20%7D%0A%20%20%20%20department%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D). We loaded up the same query as you have in the `people` folder by default. Use this interface to explore our schema and put together a query that fetches the data you need for the page.
+Our CMS, [DatoCMS](https://www.datocms.com/), exposes a public GraphQL API to our data which you can explore [here](https://cda-explorer.datocms.com/?apitoken=<ADD_DATO_API_TOKEN_HERE>&query=query%20%7B%0A%20%20allDepartments%28first%3A%20100%29%20%7B%0A%20%20%20%20name%0A%20%20%20%20parent%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%7D%0A%20%20%0A%20%20allPeople%28first%3A%20100%29%20%7B%0A%20%20%20%20name%0A%20%20%20%20avatar%20%7B%0A%20%20%20%20%20%20url%0A%20%20%20%20%7D%0A%20%20%20%20department%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D) using GraphiQL. We loaded up the same query as you have in the `people` folder by default. Use this interface to explore our schema and put together a query that fetches the data you need for the page.
 
 ### Building The Page
 
 Once you have the data, the next step is to build out the page! We expect the markup to be clear and semantic, and the styling to be clear, well-structured, and good-looking (we use [CSS Modules](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css)). Do your best to visually match the styling in the Figma design. Feel free to build out components as part of the page to reduce repetition and break out reasonable chunks of code. The `components` folder can be a home to them.
 
-Matching person results should update in real-time while typing. Checking the "Hide people missing a profile image" should filter out results that don't have an image. Use the placeholder image from the Figma file for those without an image.
```

Comment:
Just realized candidates will be able to see that this was "checking," which may be too much of a clue. :/